docs/subnets/deploying-subnets-on-prod.md

```diff
@@ -0,0 +1,187 @@
+
+### Server location
+
+You can run a node on a physical computer that you own and run, or on a cloud instance. Although running on your own HW may seem like a good idea, unless you have a sizeable DevOps 24/7 staff we recommend using cloud service providers as they generally provide reliable computing resources that you can count on to be properly maintained and monitored.
+
+#### Local servers
+
+If you plan on running nodes on your own hardware, make sure they satisfy the minimum HW specification as outlined earlier. Pay close attention to proper networking setup, making sure the p2p port (9651) is accessible and public IP properly configured on the node. Make sure the node is connected to the network physically (not over WiFi), and that the router is powerful enough to handle a couple of thousands of persistent TCP connections and that network bandwidth can accommodate at least 5Mbps of steady upstream and downstream network traffic.
+
+When installing the AvalancheGo client on the machines, unless you have a dedicated DevOps staff that will take care of node setup and configuration, we recommend using the [installer script](../nodes/build/set-up-node-with-installer.md) to set up the nodes. It will abstract most of the setup process for you, set up the node as a system service and will enable easy node upgrades.
+
+#### Cloud providers
+
+There are a number of different cloud providers. We have documents that show how to set up a node on the most popular ones:
+* [Amazon Web Services](../nodes/build/setting-up-an-avalanche-node-with-amazon-web-services-aws.md)
+* [Azure](../nodes/build/set-up-an-avalanche-node-with-microsoft-azure.md)
+* [Google Cloud Platform](../nodes/build/set-up-an-avalanche-node-with-google-cloud-platform.md)
+
+There is a whole range of other cloud providers that may offer lower prices or better deals for your particular needs, so it makes sense to shop around.
+
+Once you decide on a provider (or providers), if they offer instances in multiple data centers, it makes sense to spread the nodes geographically since that provides a better resilience and stability against outages.
+
+### Number of Validators
+
+Number of validators on a subnet is a crucial decision you need to make. For stability and decentralization, you should strive to have as many validators as possible.
+
+For stability reasons our recommendation is to have **at least** 5 full validators on your Subnet. If you have less than 5 validators your Subnet liveness will be at risk whenever a single validator goes offline, and if you have less than 4 even one offline node will halt your Subnet.
+
+Considering that at times you will have to take nodes offline, for routine maintenance (at least for node upgrades which happen with some regularity) or unscheduled outages and failures you need to be able to routinely handle at least one node being offline without your Subnet performance degrading.
```

Comment:
Do we want to recommend more than 5?

Reply:
As far as I understand, 5 is the bare minimum from a subnet stability standpoint. The more, the better, but since running nodes is a non-trivial expense (2000AVAX per validator) I would hesitate to make more than 5 an official recommendation. I will add language to ephasize that more validators is certainly better. If the platform team feels that 5 is not enough for production Subnets, we can change it. Let me know.